docs/docs/cmd/spo/site/site-set.md

```diff
@@ -129,3 +167,9 @@ Unset the logo on the site
 ```sh
 m365 spo site set --url https://estruyfdev2.sharepoint.com/sites/sales --siteLogoUrl ""
 ```
+
+Lock the site preventing users from accessing it. Wait for the configuration to complete
+
+```sh
+m365 spo site classic set --url https://contoso.sharepoint.com/sites/team --LockState NoAccess --wait
```

Comment:
I would avoid using `classic` word in the `site set` spec 🤔 ... or am I missing something 😉
could you update this small detail 👍